bioel/bioel/ontology.py

```diff
@@ -392,9 +396,44 @@ def load_ncbi_taxon(self):
     def load_csv(self):
         pass
 
-    def load_json(self):
-        pass
+    @classmethod
+    def load_json(cls, filepath, name=None):
```

Comment:
Can you also write a detailed description of what functionalities `load_json` can offer/support?